ADD-ONS-CHANGELOG.md

```diff
@@ -0,0 +1,59 @@
+# React Add-Ons Change Log
```

Comment:
Do we want to have this file long term? Since these are final versions ever (hopefully), it seems a bit redundant to add a new file that we’ll never change, for parts that were technically discontinued. Maybe we could put this up as a gist instead?